### PR TITLE
[#1636] [#1637] get color refactoring

### PR DESCRIPTION
**Pull request by aluarius**
closes: #1637 
closes: #1636
closes: #1658

## Description
- Added warning (only in development mode), if not proper color was passed in getColor function.
- Default color changed from black to primary.
- Deleted isCSSVariable check option in getColor cause of possible problems with setting opacity for the color.
- Deleted CSS.supports check option cause of SSR problems.
- Fixes in the components due to the previous two items.
- Added optional boolean prop for getColor, which will force the function to return a css variable, not hex. Сonsequently changed getColor call in colorsToCssVariables function.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by m0ksem** [comment]
Check if this PR fixes #1658

File: packages/ui/src/components/va-button-toggle/VaButtonToggle.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default defineComponent({
     options: { type: Array as PropType<any[]>, default: () => [] },
     color: { type: String as PropType<string>, default: 'primary' },
     textColor: { type: String as PropType<string>, default: undefined },
-    activeButtonTextColor: { type: String as PropType<string>, default: 'var(--va-white)' },
+    activeButtonTextColor: { type: String as PropType<string>, default: '#ffffff' },
```
Comment (m0ksem) [inline packages/ui/src/components/va-button-toggle/VaButtonToggle.vue:45]:
```suggestion
    activeButtonTextColor: { type: String as PropType<string>, default: 'white' },
```

We have white in ColorConfig's default theme, so getColor shoulw handle it 👍

File: packages/ui/src/services/color-config/color-config.ts [hunk 2]
```diff
@@ -29,25 +26,40 @@ export const getColors = (): ColorConfig => {
...
 
   if (colors[prop]) {
-    return colors[prop]
+    return preferVariables ? `var(--va-${prop})` : colors[prop]
   }
 
-  if (isColor(prop) || isCSSVariable(prop) || CSS.supports('color', prop)) {
+  /**
+   * Check options <or> isCSSVariable(prop) <or> CSS.supports('color', prop) deleted due to problems with SSR and opacity.
+   * Details are in the discussion: https://github.com/epicmaxco/vuestic-ui/pull/1589
```
Comment (m0ksem) [inline packages/ui/src/services/color-config/color-config.ts:52]: Let's better have this commet in isColor function description.

**Review by m0ksem — CHANGES_REQUESTED** [review]
Very good! 😁.
Let's have a few minor changes here.

File: packages/ui/src/components/va-button-toggle/VaButtonToggle.vue (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/components/va-button-toggle/VaButtonToggle.vue]: No, we haven't :)

**Comment by aluarius** [comment]
> Check if this PR fixes #1658

Yes, it is.

File: packages/ui/src/components/va-button-toggle/VaButtonToggle.vue (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/components/va-button-toggle/VaButtonToggle.vue]: UPD: it was decided to add white & black in default theme colors preset.

**Pull request merged**